frontend/resourceadm/pages/MigrationPage/MigrationPage.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import classes from './MigrationPage.module.css';
+import { MigrationStep } from 'resourceadm/components/MigrationStep';
+import { Checkbox, Button, TextField } from '@digdir/design-system-react';
+import { Link } from 'resourceadm/components/Link';
+import { ExternalLinkIcon } from '@navikt/aksel-icons';
+
+/**
+ * Page that shows the information about migrating from Altinn 2 to Altinn 3
+ */
+export const MigrationPage = () => {
+  const [step1Checked, setStep1Checked] = useState(false);
```

Comment:
Check if it's possible to declare states like this instead
```suggestion
  const [state, setState] = useState({
  step1Checked: false,
  step2Checked: false,
  step3Checked: false,
  beforeStartChecked: false,
  migrationDate: new Date().toISOString().split('T')[0],
  migrationTime: '00:00',
});

// Usage
<MigrationStep
  isChecked={state.step1Checked}
  onToggle={(checked) => setState({ ...state, step1Checked: checked })}
/>
```

Reply:
Thats a nice way to do it. I'll keep that in mind when the page will be implemented 100% after the final agreements on how it will work :)